### PR TITLE
BOT-2134  Remove Lato from components 🐐

### PR DESCRIPTION
**Pull request by mikrotron**
### Purpose 🚀
Make Paprika components agnostic of app-level base font.

### Notes ✏️
- Removed `font-family` css rule from `<Button>` + `<DropdownMenu.Item>`.
- Also introduced the Storybook `addon-cssresources` to allow previewing components with `Lato` or `IBM Plex`.

### Updates 📦
- [x] MINOR (backward compatible) changes to `@paprika/button`, `@paprika/dropdown-menu`

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/BOT-2134--remove-lato-from-components

### Screenshots 📸
CSS Resources Addon
cssfonts

### References 🔗
https://aclgrc.atlassian.net/browse/BOT-2134

### CODE REVIEW

File: .storybook/reset.scss [hunk 1]
```diff
@@ -11,6 +15,22 @@ body,
...
+select,
+textarea {
+  font-family: inherit;
+}
+
+body {
   color: $color--black;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-size: font-scale();
+  line-height: line-height-scale();
```
Comment (allison-c) [inline .storybook/reset.scss:35]: I think the line-height changed quite a lot in screener, for example this one https://screener.io/v2/states/H1OkdIZtV.acl-services-paprika/BOT-2134--remove-lato-from-components/1280x1024/Chrome/sortableautomation-tests-screener-0
is it ok?

File: .storybook/config.js [hunk 2]
```diff
@@ -14,9 +15,24 @@ addParameters({
   previewTabs: {
     canvas: { hidden: true },
   },
+  cssresources: [
```
Comment (alexzherdev) [inline .storybook/config.js:18]: ooh TIL

File: .storybook/reset.scss (hunk 1 above)
Comment (mikrotron) [reply .storybook/reset.scss:35]: I'm okay with it. This should now more accurately reflect our apps (they should have a default line-height of ~1.4).  In that specific case, only the dummy content is affected.

File: .storybook/reset.scss [hunk 3]
```diff
@@ -11,6 +15,22 @@ body,
...
+
+// TYPOGRAPHY
+
+// From normalize.css
+// https://github.com/necolas/normalize.css/blob/fc091cce1534909334c1911709a39c22d406977b/normalize.css#L160-L165
+button,
+input,
+select,
+textarea {
+  font-family: inherit;
```
Comment (alexzherdev) [inline .storybook/reset.scss:28]: interesting 👍

**Pull request merged**